app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Managing behaviour in the classroom"
+heading: "Managing behaviour in the classroom"
+subcategory: Qualifications and experience
+description: |-
+  Teacher training adviser Jane Wilkinson answers questions about how to manage behaviour in the classroom.
```

Comment:
I think it would be worth looking at search terms and rewording this - eg `behaviour management` is more important than `Teacher training adviser Jane Wilkinson` in terms of SEO